src/utils/checkGooglePlayServices.ts

```diff
@@ -0,0 +1,14 @@
+import { utils } from "@react-native-firebase/app"
+
+export function checkPlayServicesExample() {
+  try {
+    const result = utils().playServicesAvailability
+
+    console.log(result)
+
+    return result.isAvailable
+  } catch (err) {
+    console.error(err)
+    return false
+  }
+}
```

Comment:
questa cosa deve essere resa un pochino più furba, usando runtime require al posto di un import dentro un try-catch così che non crashi in expo go, e semplicemente fare in modo che in expo go restituisca `true` (o `null` e gestire i casi particolari, ma per lo scopo di questa funzione anche `true` va bene dato che se non siamo sicuri che non ci siano i gms vogliamo comunque tentare il rendering delle mappe)